src/Flowdock/Formatter/AbstractFlowdockFormatter.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MonologHttp\Flowdock\Formatter;
+
+abstract class AbstractFlowdockFormatter implements FlowdockFormatterInterface
+{
+    public function __construct(string $flowToken)
```

Comment:
I don't understand what we are trying to solve here.. the `$flowToken` should have been nullable on the previous PR.